types/configs/editor-config.d.ts

```diff
@@ -5,8 +5,9 @@ import {SanitizerConfig} from './sanitizer-config';
 export interface EditorConfig {
   /**
    * Element where Editor will be append
+   * TODO: rename holderId on holder, element or el in next major for better consistent
```

Comment:
It's better to add a `holder` config property right now and leave `holderId` for the backward compatibility. We will update docs and remove holderId from there. In next releases we will remove `holderId`

Reply:
Fix it